Logger/WebRequestProcessor.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * This file contains only the WebProcessorMonolog class.
+ */
+
+declare(strict_types=1);
+
+namespace Wikimedia\ToolforgeBundle\Logger;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * WebProcessorMonolog extends information included in error reporting.
```

Comment:
Should be `WebRequestProcessor `. This was my mistake in Event Metrics. Same with lines 3 and 21, though the "this file contains only..." comment block seems unnecessary to begin with (not sure where I picked up that habit).

Or you could just rename the file to `WebProcessorMonolog`. Doesn't matter :)

Reply:
Oops! Fixed. I got rid of the whole constructor docblock because it wasn't saying anything else.